libsel4/include/interfaces/sel4.xml

```diff
@@ -1452,7 +1452,7 @@
                 If the object is a notification and it is bound to the scheduling context, unbind it.
             </brief>
             <description>
-                See <autoref label="sec:passive"/>
+                <docref>See <autoref label="sec:passive"/></docref>
             </description>
```

Comment:
These will then be empty `<description>` tags. Have you had a look at the API docs output if that works out Ok?

Reply:
@lsf37 
The description gets removed in parse_doxygen.xml:
```
def generate_api_doc(self, level, member, params, ret, details, errors):
        manual_node = member.manual

        # Descriptions that just contain a document reference are removed.
        # Found by the 'DOCREF' symbol
        match_details = re.match(r'^DOCREF$', details, re.M | re.I)
        if match_details:
            details_string = ""
    ```